docs/src/examples/BreadcrumbExample/BreadcrumbExampleFirst.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Breadcrumb } from '../../../../src/components';
+import { ISourceData } from '../../../../src/components/Breadcrumb/Breadcrumb';
+import * as styles from '../../styles/components-page.scss';
+
+const breadcrumbData: ISourceData[] = [
+  {
+    name: 'Home',
+    type: 'active',
+  }, {
+    name: 'Home1',
+    type: 'active'
+  }, {
+    name: 'Home2',
+    type: 'active'
+  }, {
+    name: 'Home3',
+    type: 'active'
+  },
+];
+
+class BreadcrumbExample extends React.Component {
+
+  BreadcrumbClick = () => {
+    console.log('Breadcrumb clicked...');
+  }
+
+  render() {
+    return (
+      <div className={styles.example}>
+        <Breadcrumb onBreadcrumbClick={this.BreadcrumbClick} direction={'left'} source={breadcrumbData} displayStyle={'primary'}/>
```

Comment:
Instead of having `onBreadcrumbClick={this.BreadcrumbClick}` in here it would be better to have the callback function in  `breadcrumbData` so it will be like `{ name: 'xyz', type: 'active', action: () => { console.log('xyz is clicked') }}`, then in Breadcrumb component you will call the action callback like `this.props.source.action()`. This way user can do anything they want on click

Reply:
@HiirenP you can directly write displayStyle like this `displayStyle="primary"`

Reply:
resolved